zammad/ci/full-values.yaml

```diff
@@ -87,3 +87,6 @@ podLabels:
 
 podAnnotations:
   my-pod-annotation: my-pod-annotation-value
+
+podSpec:
+  activeDeadlineSeconds: 1337
```

Comment:
I believe this must be added to the `initJob` section, just as in values.yaml.

Reply:
@mgruner oh yes, you are right, sorry this was my fault.